canned/mysql/mysql.go

```diff
@@ -0,0 +1,113 @@
+package canned
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/pkg/errors"
+	testcontainers "github.com/testcontainers/testcontainers-go"
+)
+
+const (
+	DefaultImageVersion = "mysql:8.0"
+)
+
+type MySQLContainerRequest struct {
+	testcontainers.GenericContainerRequest
```

Comment:
I think we should not accept a request here, we should build it based on what we know from the image the other params we get from the user. What do you think?